resources/views/main/about.blade.php

```diff
@@ -73,6 +73,9 @@
                         </ul>
                     </div>
                     <div class="main-menu-wrapper__right">
+                        <div class="main-menu-wrapper__call">
+                            <a href="https://{{ config('app.domains.app') }}/login" class="thm-btn welcome-two__btn">Login</a>
+                        </div>
                         <div class="main-menu-wrapper__call">
                             <a href="https://{{ config('app.domains.app') }}/register" class="thm-btn welcome-two__btn">Free Demo</a>
                         </div>
```

Comment:
Why we are using config for get domain name since we have already declared route for login and register?